src/components/reminder-form/reminder-form.tsx

```diff
@@ -3,33 +3,56 @@ import React, { FormEvent, useState } from "react";
 
 const Form = ({ }: {}) => {
   const [taskName, setTaskName] = useState('');
-  const [recurring, setRecurring] = useState(false);
+  const [taskMessage, setTaskMessage] = useState('');
+  const [reminderType, setReminderType] = useState('');
+  const [reminderDate, setReminderDate] = useState('');
+  const [recurring, setRecurring] = useState(true);
+  const [reccuringFrequency, setReccuringFrequency] = useState('');
+  const [reminderBeforeMs, setMs] = useState('');
+  let task = []
   const handleSubmit = (e: FormEvent<HTMLFormElement>): void => {
     e.preventDefault();
+    let taskObject = {
+      taskName: taskName,
+      reminderDate: reminderDate,
+      recurring: recurring,
+      recurranceConfig: {
+        reccuringFrequency: reccuringFrequency,
+      },
+      reminderConfig: {
+        customeMessage: taskMessage,
+        reminderBeforeMs: reminderBeforeMs,
+        reminderType: reminderType,
+      }
+    }
+    console.log(taskObject);
```

Comment:
Please, no consoles in the code. If you need these to be printed, use proper logger we can redirect logs somewhere else from the browser with